public/locales/en/premium-video.json

```diff
@@ -0,0 +1,5 @@
+{
+    "errors": {
+        "missing-password": "You need to provide a password within the link in order to join a room"
```

Comment:
Why not something like `The provided URL is missing the password, ask the organizer another one`

Reply:
missing french transl